app/views/campaign_pages/edit.slim

```diff
@@ -1,4 +1,4 @@
-= form_for @campaign_page do |f|
+= form_for @campaign_page, remote: true do |f|
```

Comment:
this breaks the validation errors and the successful save notifications. I don't think a page refresh is bad user experience, but I do think not being able to tell if your work saved is. given that we'd have to do more to get that working again with javascript, why not just leave `remote: false` for the MVP?